override/JSON.java

```diff
@@ -1,54 +1,37 @@
-/**
- * sis-wag
- * Serves sis-data
+/*
+ * Clever API
+ * The Clever API
+ *
+ * OpenAPI spec version: 2.0.0
  *
- * OpenAPI spec version: 1.2.0
- * 
  *
  * NOTE: This class is auto generated by the swagger code generator program.
  * https://github.com/swagger-api/swagger-codegen.git
  * Do not edit the class manually.
```

Comment:
Just to make sure I understand, the base of this file is autogenerated but we've manually tweaked it to handle events? (I'm assuming it's been tweaked since it's in the `override/` directory)

Reply:
Yeah we copied the base file and made some tweaks to handle events